<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QWidget" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>721</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Player">
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Volume">
        <property name="text">
         <string>音量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Open">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Jpeg">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>520</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>JPEG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>编码的帧</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>视频文件路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_show">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_key">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>520</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>关键帧</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>质量因子</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>190</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomSlider</class>
   <extends>QSlider</extends>
   <header location="global">customslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
